spec.html

```diff
@@ -30298,12 +30298,12 @@ <h1>Atom</h1>
               1. Let _xe_ be _x_'s _endIndex_.
               1. Let _ye_ be _y_'s _endIndex_.
               1. If _direction_ is equal to +1, then
-                1. Assert: _xe_ &lte; _ye_.
-                1. Let _s_ be a fresh List whose characters are the characters of _Input_ at indices _xe_ (inclusive) through _ye_ (exclusive).
+                1. Assert: _xe_ &le; _ye_.
```

Comment:
I think the intention here was less than or equal to. While gte/lte are not entities, `&le;=` would preserve the intent. Is < a better assert? /cc @littledan

Reply:
never mind I don't know HTML entities.